sellercenter-category/src/main/java/com/gfg/sellercenter/category/service/CategoryFulfillmentVisibilityService.java

```diff
@@ -0,0 +1,82 @@
+package com.gfg.sellercenter.category.service;
+
+import com.gfg.sellercenter.category.entity.CategoryDetails;
+import com.gfg.sellercenter.category.entity.CategoryFulfillmentSetting;
+import com.gfg.sellercenter.category.entity.CategoryStatus;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class CategoryFulfillmentVisibilityService {
+    private static final boolean FULFILLMENT_ENABLED = true;
+    private static final boolean FULFILLMENT_DISABLED = false;
+
+    public List<Integer> getListOfFulfilmentEnabledCategoriesBySeller(Map<Integer, CategoryDetails> categories, Map<Integer, CategoryFulfillmentSetting> fulfillmentSettings) {
+        // enable fulfillment in all categories and subcategories where specifically that seller has fulfillment enabled
```

Comment:
Can we try to avoid comments inside of a method?

Can we also split this big method into smaller pieces?

Reply:
Changed according your suggestions